script/deploy-stanup-demo

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -euxo pipefail
+
+cd $GITHUB_WORKSPACE/demos/serverless
+npm ci
+
```

Comment:
Maybe we can export the main aws creds here, the same way we do for the deploy gamma and deploy beta. We might not need the configure AWS creds step in that case I think.

Reply:
fixed